public/scripts/todoListController.js

```diff
@@ -0,0 +1,18 @@
+angular
+  .module('MyTodoApp')
+  .controller('TodoListController', ['TodoService', function(TodoService) {
+
+    this.todoList = TodoService.todoList;
+
+    this.completedList = TodoService.completedList;
+
+    this.addTask = function(newTask) {
```

Comment:
Here you're using the controller as a wrapper for a service - which is *fine* - but it doesn't really enhance flexibility or extensibility or anything b/c the service basically does exactly what the controller should do.